tests/helpers.ts

```diff
@@ -86,7 +86,11 @@ export const fromSigner = (contract: Contract, address: string): Contract => {
   return patchContractMethods(contract.connect(address))
 }
 
-export const bnArg = (value: number | string | number[] | Uint8Array | Buffer | BN, length = 32) =>
-  new BN(value, undefined, 'le').toArray('le', length)
+export const bnArg = (value: number | string | number[] | Uint8Array | Buffer | BN, len = 32) => {
+  if (len == 1) {
```

Comment:
It is not a good solution=) Because function should return `BN`. Better to re-write test to use `1`

Reply:
agreed